### PR TITLE
docs: visual studio removed from nav and updated to refer to getting started

### PR DESCRIPTION
**Pull request by jenniferfell**
…instead of quickstart repo

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Visual Studio 2015 QuickStart https://next.angular.io/guide/visual-studio-2015

* Is in navigation under Setup & Deployment ? Dev Tool Integration.
* Refers to old quickstart app and repo
* Has old TypeScript 2.2 dependency
* Comes up prominently in searches for "quickstart"

Issue Number: N/A

## What is the new behavior?

Temporary change until we can update this doc to include more versions of Visual Studio: 

Visual Studio 2015 QuickStart https://next.angular.io/guide/visual-studio-2015

* Is hidden from navigation 
* Page title is changed to "Using Angular with Visual Studio 2015" to make search more accurate
* Refers to Getting Started for info about node and npm versions and installation
* Refers to Getting Started to download starter app instead of old quickstart repo
* Has new TypeScript 3.1 dependency

## Does this PR introduce a breaking change?
```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by jenniferfell** [comment]
Note: This doc update refers readers to "Final code" section of Getting Started to download sample app. That heading will appear when this other PR lands: https://github.com/angular/angular/pull/26093

**Comment by jenniferfell** [comment]
I'll need to rebase after https://github.com/angular/angular/pull/26375 is merged

**Comment by jenniferfell** [comment]
I left the "Dev Tool Integration" subheading with only one child, because:

* It gives context to the Language Services title, which isn't an obvious title to all users
* I think we can get VS doc back soon, so I don't think the single child situation will exist for long

But I'm willing to remove that subheading if you want. :-)

File: aio/content/guide/visual-studio-2015.md [hunk 1]
```diff
@@ -100,35 +92,33 @@ find them in either location, it will use its own versions of the tools.
...
+To install TypeScript 3.1:
 
- * Download and install **[TypeScript 2.2 for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48593)**
+ * Download and install [TypeScript 3.1 for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48593),
 
- * OR install it with npm: `npm install -g typescript@2.2`.
+ * OR install it with npm: `npm install -g typescript@3.1`.
 
-You can find out more about TypeScript 2.2 support in Visual studio **[here](https://blogs.msdn.microsoft.com/typescript/2017/02/22/announcing-typescript-2-2/)**.
+You can find out more about TypeScript 3.1 support in Visual Studio **[here](https://blogs.msdn.microsoft.com/typescript/2018/09/13/announcing-typescript-3-1-rc/)**.
```
Comment (IgorMinar) [inline aio/content/guide/visual-studio-2015.md]: use the 3.1 final announcement link instead: https://blogs.msdn.microsoft.com/typescript/announcing-typescript-3-1/

File: aio/content/navigation.json [hunk 2]
```diff
@@ -541,8 +541,9 @@
            },
            {
              "url": "guide/visual-studio-2015",
-             "title": "Visual Studio 2015 QuickStart",
-             "tooltip": "Use Visual Studio 2015 with the QuickStart files."
+             "title": "Visual Studio 2015",
+             "tooltip": "Use Visual Studio 2015 with the QuickStart files.",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: maybe use the title for consistency: "Using Angular with Visual Studio 2015"

File: aio/content/navigation.json [hunk 3]
```diff
@@ -541,8 +541,9 @@
            },
            {
              "url": "guide/visual-studio-2015",
-             "title": "Visual Studio 2015 QuickStart",
-             "tooltip": "Use Visual Studio 2015 with the QuickStart files."
+             "title": "Visual Studio 2015",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: I thought we said that we wanted to remove this doc from the nav, but leave it for people that have a link to it already. Or am I mistaken?

File: aio/content/guide/visual-studio-2015.md (hunk 1 above)
Comment (jenniferfell) [reply aio/content/guide/visual-studio-2015.md]: Thanks! done

File: aio/content/navigation.json (hunk 2 above)
Comment (jenniferfell) [reply aio/content/navigation.json]: Done, but no one will see it :-)

File: aio/content/navigation.json (hunk 3 above)
Comment (jenniferfell) [reply aio/content/navigation.json]: Yup. Done. The `"hidden": true` property on the nav item hides it, so it doesn't appear in the navigation on the left but keeps the file in the AIO build.

File: aio/content/guide/visual-studio-2015.md [hunk 4]
```diff
@@ -160,12 +150,10 @@ no authentication, and no hosting. Pick the template and options appropriate for
 
 
 <h2 id='copy'>
-  Step 3: Copy the QuickStart files into the ASP.NET project folder
+  Step 3: Copy the Angular project files into the ASP.NET project folder
```
[the remark this answers is not in the document]
Comment (jenniferfell) [reply aio/content/guide/visual-studio-2015.md]: done

**Comment by jenniferfell** [comment]
@gkalpak @IgorMinar Thanks for reviewing. Changes are incorporated. I just need one of you to approve on behalf of marketing group, and then we can mark for merge. Thx!

**Review by IgorMinar — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**